dlrover/python/master/elastic_training/rdzv_manager.py

```diff
@@ -155,26 +162,32 @@ def _check_rdzv_completed(self):
             self._lastcall_time = 0
             self._log_rendezvous_info()
             if self._waiting_nodes:
-                waiting_node_ranks = sorted(list(self._waiting_nodes.keys()))
+                waiting_node_ids = []
+                for node in self._waiting_nodes.values():
+                    waiting_node_ids.append(node.node_id)
                 logger.warning(
                     f"Waiting nodes not in {self._rdzv_round} rendezvous "
-                    f"are {waiting_node_ranks}."
+                    f"are {waiting_node_ids}."
                 )
         elif time.time() - self._latest_log_nodes_time > 60:
             self._latest_log_nodes_time = time.time()
-            waiting_node_ranks = sorted(list(self._waiting_nodes.keys()))
-            logger.info(
-                f"Waiting nodes in rendezvous are {waiting_node_ranks}"
-            )
+            waiting_nodes = []
+            for _, node in self._waiting_nodes.items():
+                waiting_nodes.append(node.node_id)
+            logger.info(f"Waiting nodes in rendezvous are {waiting_nodes}")
         return rdzv_completed
 
     def _log_rendezvous_info(self):
-        node_ranks = sorted(list(self._rdzv_nodes.keys()))
+        node_ids = []
+        for node in self._rdzv_nodes.values():
+            node_ids.append(node.node_id)
+        node_ids = sorted(node_ids)
+        node_rdzv_times = self._map_node_rank_to_id(self._node_rdzv_times)
         logger.info(
             f"Completed {self._rdzv_round} round "
-            f"rendezvous of {self._name} is {node_ranks} \n"
+            f"rendezvous of {self._name} is {node_ids} \n"
```

Comment:
Better print out the node id and rank in the format of 'id:rank'?

Reply:
`rank: id` may be better because we usually need to find the pod with rank 0.